### PR TITLE
feat(debugFileDetails): Add tooltips

### PR DESCRIPTION
**Pull request by priscilawebdev**
Implemented tooltip descriptions:

image

### CODE REVIEW

**Comment by priscilawebdev** [comment]
@robinrendle 

1. should we use the hovercard component for the table description? it would be like: 
image

2. At the moment, we use tooltips with the styles you used in Figma only in the graphics. Should we update the styles?

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/panel.tsx [hunk 1]
```diff
@@ -99,89 +104,96 @@ function Panel({
...
+        <QuestionTooltip
+          title={description}
+          size="xs"
+          position="top"
+          isHoverable
+          popperStyle={css`
+            @media (min-width: ${theme.breakpoints[0]}) {
+              max-width: 350px;
+            }
+          `}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/panel.tsx:116]: Hmm are you sure this works? You may need to use `import {ClassNames} from '@emotion/core';`

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/panel.tsx:116]: yes it works :)

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx [hunk 2]
```diff
@@ -33,32 +33,32 @@ type Props = AsyncComponent['props'] &
...
     const {organization, projectId, image} = this.props;
     const {debug_id, candidates = []} = image;
+    const {builtinSymbolSources} = this.state || {};
 
     const uploadedDebugFiles = this.getUplodedDebugFiles(candidates);
+    const endpoints: ReturnType<AsyncComponent['getEndpoints']> = [];
 
-    if (!uploadedDebugFiles) {
-      return [];
-    }
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx:58]: Is it possible to not have uploaded debug files *and* have builtin symbol sources?

**Review by billyvg — APPROVED** [review]
This would be a good candidate to have in storybook as I wouldn't know how to test this in Sentry.

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx:58]: yes it is

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx:58]: yes it is

**Pull request merged**